diff --git a/node_modules/@types/react-bootstrap-table2-toolkit/index.d.ts b/node_modules/@types/react-bootstrap-table2-toolkit/index.d.ts
index 7288399..c34823c 100644
--- a/node_modules/@types/react-bootstrap-table2-toolkit/index.d.ts
+++ b/node_modules/@types/react-bootstrap-table2-toolkit/index.d.ts
@@ -7,7 +7,7 @@
 // documentation taken from https://react-bootstrap-table.github.io/react-bootstrap-table2/docs/table-props.html
 
 import { CSSProperties, ReactNode } from 'react';
-import { ColumnDescription } from 'react-bootstrap-table-next';
+import { ColumnDescription, SearchProps } from 'react-bootstrap-table-next';
 
 /**
  * declaration for table toolkit sub module
@@ -16,6 +16,7 @@ export interface InjectedSearchProps {
     searchText: string;
     onSearch: (val: string) => void;
     onClear: () => void;
+    className?: string | undefined;
 }
 
 export interface SearchMatchProps<T extends object = any> {
@@ -109,15 +110,14 @@ export interface ExportCSVButtonProps {
 export namespace CSVExport {
     function ExportCSVButton(props: ExportCSVButtonProps): React.ReactElement | null;
 }
-
-export interface SearchBarProps {
-    onSearch: (searchText: string) => void;
+// global-scoreboard\src\Components\GameSearch\GameCategorySearchBar.tsx
+export interface SearchBarProps<T = any> extends SearchProps<T> {
     className?: string | undefined;
-    placeholder?: string | undefined;
     style?: CSSProperties | undefined;
     delay?: number | undefined;
     searchText?: string | undefined;
     tableId?: string | undefined;
+    ref?: RefObject<Component<SearchProps<T>, {}, any>>;
 }
 export interface ClearSearchButtonProps {
     onClear?: (() => void) | undefined;
