diff --git a/node_modules/react-bootstrap/esm/types.d.ts b/node_modules/react-bootstrap/esm/types.d.ts
index d047629..fce5440 100644
--- a/node_modules/react-bootstrap/esm/types.d.ts
+++ b/node_modules/react-bootstrap/esm/types.d.ts
@@ -1,6 +1,6 @@
 /// <reference types="react" />
 import PropTypes from 'prop-types';
-export declare type Variant = 'primary' | 'secondary' | 'success' | 'danger' | 'warning' | 'info' | 'dark' | 'light' | string;
+export declare type Variant = 'primary' | 'secondary' | 'success' | 'danger' | 'warning' | 'info' | 'dark' | 'light' | (string & { fromT?: any });
 export declare type ButtonVariant = Variant | 'link' | 'outline-primary' | 'outline-secondary' | 'outline-success' | 'outline-danger' | 'outline-warning' | 'outline-info' | 'outline-dark' | 'outline-light';
 export declare type Color = 'primary' | 'secondary' | 'success' | 'danger' | 'warning' | 'info' | 'dark' | 'light' | 'white' | 'muted';
 export declare type Placement = import('react-overlays/usePopper').Placement;
